ray-curator/ray_curator/stages/image/embedders/clip_embedder.py

```diff
@@ -0,0 +1,82 @@
+from dataclasses import dataclass
+
+import torch
+from loguru import logger
+
+from ray_curator.backends.base import WorkerMetadata
+from ray_curator.models.clip import CLIPImageEmbeddings
+from ray_curator.stages.base import ProcessingStage
+from ray_curator.stages.resources import Resources
+from ray_curator.tasks import ImageBatch
+
+
+@dataclass
+class ImageEmbeddingStage(ProcessingStage[ImageBatch, ImageBatch]):
+    """Stage for generating image embeddings using CLIP model.
+
+    This class processes image batches through a CLIP model to generate
+    embeddings for each image. It assumes image data is already loaded
+    in ImageObject.image_data and stores embeddings in ImageObject.embedding.
+    """
+    model_dir: str = "models/clip"
+    num_gpus_per_worker: float = 0.25
+    model_batch_size: int = 32  # Number of images to process through model at once
```

Comment:
```suggestion
    model_inference_batch_size: int = 32  # Number of images to process through model at once
```
Nit: Our text classification implementation calls it `model_inference_batch_size`, maybe we should try to match?

Reply:
Fixed.